watt-aws-update.sh

```diff
@@ -61,3 +61,33 @@ if [ $? -ne 0 ]; then
 fi
 
 echo -e "\033[0;32mSuccessfully pushed watt image to \033[0m"$watt_docker_aws_repository_uri
+
+# Install jq package.
+if ! dpkg -s jq; then
+    if ! sudo apt-get install -y jq; then
```

Comment:
What about just checking if package jq is installed.
If not just print command to install and exit. And such check should be made at the script beginning.

Reply:
Done.